qutebrowser/browser/webengine/notification.py

```diff
@@ -679,7 +679,7 @@ def from_list(cls, capabilities: List[str]) -> "_ServerCapabilities":
 def _as_uint32(x: int) -> QVariant:
     """Convert the given int to an uint32 for DBus."""
     variant = QVariant(x)
-    successful = variant.convert(QVariant.Type.UInt)
+    successful = variant.convert(QMetaType(QMetaType.Type.UInt.value))
```

Comment:
This is extra awkward, QVariant.convert() takes a QMetataType which takes the value of a QMetaType.Type.